.github/CONTRIBUTING.md

```diff
@@ -48,28 +60,28 @@ You can update existing or add new topics in their respective Magento 2 versione
 
 The following guidelines may answer most of your questions and help you get started:
 
-1.  Check [existing pull requests](https://github.com/magento/devdocs/pulls) and make sure you are not duplicating work!
-1.  For large contributions or changes that include multiple files, [open an issue](#report-an-issue) and discuss it with us first. This may further prevent duplicate or unnecessary effort.
-1.  [Chunk many small/medium changes](#tips-for-writing-content) into one or two PRs. This helps us to efficiently and effectively facilitate your contribution.
-1.  Familiarize yourself with the existing documentation. Look through and search the guides to decide where to add your topics.
+1. Check [existing pull requests](https://github.com/magento/devdocs/pulls) and make sure you are not duplicating work!
+1. For large contributions or changes that include multiple files, [open an issue](#report-an-issue) and discuss it with us first. This may further prevent duplicate or unnecessary effort.
+1. [Chunk many small/medium changes](#tips-for-writing-content) into one or two PRs. This helps us to efficiently and effectively facilitate your contribution.
+1. Familiarize yourself with the existing documentation. Look through and search the guides to decide where to add your topics.
 
-    -   The DevDocs team can find the best home for your new topics and add it to the navigation.
-    -   If a topic has a symlink, you can remove it with Git commands and add a new file. Copy and paste a previous version of the topic to get started.
+    - The DevDocs team can find the best home for your new topics and add it to the navigation.
+    - If a topic has a symlink, you can remove it with Git commands and add a new file. Copy and paste a previous version of the topic to get started.
```

Comment:
Will symlinks still exist in the new versioned devdocs?

Reply:
I suspect many docs will need revamping when we move fully to multi-repo, multi-branch.